src/app/GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -99,7 +100,8 @@ public RevisionDataGridView()
             InitializeComponent();
             DoubleBuffered = true;
 
-            _alternatingRowBackgroundBrush = new SolidBrush(KnownColor.Window.MakeBackgroundDarkerBy(0.025)); // 0.018
+            _rowBackgroundBrush = new SolidBrush(AppColor.PanelBackground.GetThemeColor());
```

Comment:
Maybe this color should be separate.
For dark mode, a tuning is needed and I believe it looks better.
For light+ the change should maybe be more subtle.
There are three additional colors now, a fourth is not so bad...